fuzz/src/main/java/tech/pegasys/teku/fuzz/input/BlockHeaderFuzzInput.java

```diff
@@ -28,10 +29,11 @@
 public class BlockHeaderFuzzInput
     extends Container2<BlockHeaderFuzzInput, BeaconState, BeaconBlock> {
 
+  @SuppressWarnings("unchecked")
   public static ContainerSchema2<BlockHeaderFuzzInput, BeaconState, BeaconBlock> createType(
       final SpecVersion spec) {
     return ContainerSchema2.create(
-        spec.getSchemaDefinitions().getBeaconStateSchema(),
+        (SszSchema<BeaconState>) spec.getSchemaDefinitions().getBeaconStateSchema(),
```

Comment:
It's a shame to need these casts popping up in a bunch of places.  I failed at working out how to avoid them though..

It may be worth having a `SszSchema.as` like:
```
  static <X extends SszData> SszSchema<X> as(final Class<X> clazz, final SszSchema<?> schema) {
    return (SszSchema<X>)schema;
  }
```

and then this would be:
```suggestion
        SszSchema.as(BeaconState.class, spec.getSchemaDefinitions().getBeaconStateSchema()),
```

Not entirely sure its better but means we don't need `@SuppressWarnings("unchecked")` at each call site - just once on the `as` function.

Or possibly just a getter in `SchemaDefinitions` that can give a "basic" `SszSchema<BeaconState>`?

Reply:
Went with the static `SszSchema.as` helper - that cleans things up a little bit.